dataobj/schedule_entry.h

```diff
@@ -70,6 +73,11 @@ struct schedule_entry_t
 	uint16 waiting_time_shift;
 	
 	uint16 spacing, spacing_shift, delay_tolerance;
+
+	/**
+	 * Overwrite max speed of convoy at this stops.
+	 */
+	uint16 max_speed_of_convoi;
```

Comment:
Simutrans においてはkm/hと別でspeedの単位がある（`convoi_t`あたりを見てください）ので、km/hであることを明示しないとどちらなのかよくわからなくなります。

```suggestion
	uint16 max_speed_kmh_of_convoi;
```

Reply:
kmhを明示的に付けるべき旨、承知しました。
こちら、既存のスケジュール側の最高速度にもkmhを付けるべきでしょうか？

Reply:
単位がkmhの変数についてはkmhを名前に付けてください